modules/abi.js

```diff
@@ -11,42 +12,41 @@ function isHexType(type) {
 }
 
 function padLeft(string, chars) {
-    return (new Array(chars - string.length + 1).join('0')) + string;
-};
+    return (new Array((chars - string.length) + 1).join('0')) + string;
+}
 
-ipc.on('backendAction_decodeFunctionSignature', (event, signature, data) => {
-    let paramTypes;
-    data = data.slice(10, data.length);
-    signature = signature.match(/\((.+)\)/i);
+ipc.on('backendAction_decodeFunctionSignature', (event, _signature, _data) => {
+    const data = _data.slice(10, _data.length);
+    const signature = _signature.match(/\((.+)\)/i);
 
     if (!signature) return;
 
-    paramTypes = signature[1].split(',');
+    const paramTypes = signature[1].split(',');
 
     try {
         const paramsResponse = abi.rawDecode(paramTypes, new Buffer(data, 'hex'));
-        let paramsDictArr = [];
+        const paramsDictArr = [];
 
-		// Turns addresses into proper hex string
-		// Turns numbers into their decimal string version
+        // Turns addresses into proper hex string
+        // Turns numbers into their decimal string version
         paramTypes.forEach((type, index) => {
-            let conversionFlag = isHexType(type) ? 'hex' : null,
-                prefix = isHexType(type) ? '0x' : '';
+            const conversionFlag = isHexType(type) ? 'hex' : null;
+            const prefix = isHexType(type) ? '0x' : '';
 
             paramsResponse[index] = paramsResponse[index].toString(conversionFlag);
 
-            let res = type.match(/bytes(\d+)/i);
-            if(type === 'address')
-            	paramsResponse[index] = padLeft(paramsResponse[index], 40);
-            else if(res)
-            	paramsResponse[index] = padLeft(paramsResponse[index], Number(res[1])*2);
+            const res = type.match(/bytes(\d+)/i);
+            if (type === 'address') {
+                paramsResponse[index] = padLeft(paramsResponse[index], 40);
+            } else if (res) {
+                paramsResponse[index] = padLeft(paramsResponse[index], Number(res[1]) * 2);
+            }
 
-            paramsDictArr.push({type: type, value: prefix + paramsResponse[index]});
+            paramsDictArr.push({ type, value: prefix + paramsResponse[index] });
```

Comment:
This is valid in ES 6 -> [Literal Property Value Shorthand](https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand). One may specify only the literal if key and value are similar.